src/cuda/image_function_cuda.cu

```diff
@@ -239,6 +239,59 @@ namespace
         }
     }
 
+    __global__ void rotateCuda( const uint8_t * in, uint32_t rowSizeIn, uint32_t inHeight, float inXStart, float inYStart, 
+                                uint8_t * out, uint32_t rowSizeOut, uint32_t outHeight, 
+                                float cosAngle, float sinAngle)
+    {
+        uint32_t outX = blockDim.x * blockIdx.x + threadIdx.x,
+                 outY = blockDim.y * blockIdx.y + threadIdx.y;
```

Comment:
Please use an explicit declaration of variable type in all cases as people might read that this is a single variable.

```cpp
const uint32_t outX = blockDim.x * blockIdx.x + threadIdx.x,
const uint32_t outY = blockDim.y * blockIdx.y + threadIdx.y;
```

Please change the same for below cases as well.

Reply:
Okay, done.